src/verifier.js

```diff
@@ -285,7 +285,7 @@ function verify(
       (min === 0 ||
         (now < min && value.code === 'FAST_JWT_INACTIVE') ||
         (now >= min && value.code !== 'FAST_JWT_INACTIVE')) &&
-      (max === 0 || now <= max)
+      (max === 0 || now < max)
```

Comment:
wait, if we set it to -1, why would we need to change the code?

Reply:
I changed it as I was investigating on the possible reason, it doesn't solve the issue but I'm not sure it's the correct behaviour.
if `now = max` is it still considered valid?

Reply:
Debatable but I wouldn't change the existing semantics